changelogs/fragments/3667-ldap_search.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - ldap-search - Allow community.general.ldap_search to be used even in check mode(https://github.com/ansible-collections/community.general/issues/3619)
```

Hi @sabman3 couple of minor adjustments:
```suggestion
  - ldap-search - allow it to be used even in check mode (https://github.com/ansible-collections/community.general/issues/3619).

```

---

No need to refer to the modue itself. That fragment will later become something like: https://github.com/ansible-collections/community.general/blob/stable-4/CHANGELOG.rst
It will be abundantly clear that it refers to the c.g collection.

---

/other than these minor adjustments, it looks good to me.